CHANGELOG.md

```diff
@@ -12,28 +12,30 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
```

Comment:
I think most of these changes should be reverted

Reply:
So for this, there was a ton of issues with resolving the merge conflicts on the CHANGELOG. There were repeats of a lot of stuff. I ended up having to manually comb through and make sure we only kept the change to the most recent version. I then reordered the changes to be in order of PR number while I was doing it. 

If you let me know which changes you want swapped back, I can check to see if I messed something up.

Reply:
For instance `azure-storage-common` was listed as being bumped from 12.18.0 to 12.18.1 but also from 12.18.1 to 12.19.3. So I just kept the latter since it didn't make sense to list a middle version. I will go through and double check each individually again and swap cases to show the earliest to the latest.

Reply:
I think you can just revert this whole file and only add in the line for these changes

Reply:
Sounds good